### PR TITLE
Fix bug 1343200: Update CA Cert Policy

### PR DESCRIPTION
**Pull request by pmac**

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html [hunk 1]
```diff
@@ -29,33 +32,46 @@ <h1 class="title-shadow-box">Mozilla CA Certificate Policy</h1>
...
     to ask users for further permission or information.</p>
   <p>This is the official Mozilla policy for CA certificates that are distributed
-    with Mozilla software products. This policy consists of the following three
+    with Mozilla software products. This policy consists of the following
     sections:</p>
   <ul>
     <li>
-      <p><a href="#inclusion">Applying for Inclusion of Root Certificates in Mozilla Products</a>
-      </p>
+      <p><a href="audit-requirements">Audit Requirements</a></p>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html]: Missing anchor `#` here?

File: bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html [hunk 2]
```diff
@@ -66,19 +82,184 @@ <h1 class="title-shadow-box">Mozilla CA Certificate Policy</h1>
...
     after consulting with the public Mozilla community, in order to ensure that
     all views are taken into account.</p>
-  <h2 id="inclusion">Applying for Inclusion of Root Certificates in Mozilla Products</h2>
-  <p>This section of the Mozilla CA Certificate Policy
-    describes the obligations of Certification Authorities applying for
-    inclusion of their root certificates in Mozilla Products. This includes
-    considerations that are taken into account such as the CA’s publicly
-    available documentation about their policies, and audits of the CA’s
-    operations in support of the documented policies.</p>
+  <h2>Audit Requirements</h2>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html]: Needs an ID of `audit-requirements`?

File: bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html [hunk 3]
```diff
@@ -66,19 +82,184 @@ <h1 class="title-shadow-box">Mozilla CA Certificate Policy</h1>
...
+    </li>
+    <li>the point in time date, for those that are for a point in time;</li>
+    <li>the date the report was issued (which will necessarily be after the end
+      date or point in time date).
+    </li>
+  </ol>
+  <p>For audit reports which are being supplied to maintain a certificate within
+    the Mozilla root program, they must be provided to Mozilla within three
+    months of the point in time date or the end date of the period.</p>
+  <h2>Applying for Inclusion of Root Certificates in Mozilla Products</h2>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html]: This needs an ID too?

File: bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html [hunk 4]
```diff
@@ -484,7 +536,7 @@ <h2 id="inclusion">Applying for Inclusion of Root Certificates in Mozilla Produc
         who will make a final decision.</p>
     </li>
   </ol>
-  <h2 id="maintenance">Maintaining Confidence in Included Root Certificates</h2>
+  <h2>Maintaining Confidence in Included Root Certificates</h2>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html]: This needs an ID too?

File: bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html [hunk 5]
```diff
@@ -29,33 +32,46 @@ <h1 class="title-shadow-box">Mozilla CA Certificate Policy</h1>
...
+        Inclusion of Root Certificates in Mozilla Products</a></p>
       <p>This section describes the obligations of Certification Authorities
         applying for inclusion of their root certificates in Mozilla Products.
         This includes considerations that are taken into account such as the CA’s
         publicly available documentation about their policies, and audits of the
         CA’s operations in support of the documented policies.</p>
     </li>
     <li>
-      <p><a href="#maintenance">Maintaining Confidence in Included Root Certificates</a></p>
+      <p><a href="#maintaining-confidence-in-included-root-certificates">Maintaining Confidence in
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html]: These anchor links are pretty long, suggest something shorter like it used to be i.e. `#maintenance`?

File: bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html [hunk 6]
```diff
@@ -716,7 +755,7 @@ <h2 id="maintenance">Maintaining Confidence in Included Root Certificates</h2>
       </ul>
     </li>
   </ol>
-  <h2 id="enforcement">Enforcing the Mozilla CA Certificate Policy</h2>
+  <h2>Enforcing the Mozilla CA Certificate Policy</h2>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html]: Missing ID here too

File: bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html [hunk 7]
```diff
@@ -779,4 +818,8 @@ <h2 id="enforcement">Enforcing the Mozilla CA Certificate Policy</h2>
         US-CERT organization of the U.S. Department of Homeland Security.</p>
     </li>
   </ol>
+  <hr>
+  <p>Any copyright in this document is
+    <a href="http://creativecommons.org/publicdomain/zero/1.0/">dedicated to the Public Domain</a>.
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html]: Nit: this can be `https`

**Comment by pmac** [comment]
I can fix this in the HTML, but perhaps some of these things can/should be fixed in the Markdown source as well?

/cc @gerv

**Comment by gerv** [comment]
Please fix them in the HTML; the next version of the policy removes those anchors entirely, so it's not worth fixing in the Markdown. Thanks!

**Comment by pmac** [comment]
I believe I've addressed your concerns @alexgibson. Please have another look when you can. And thanks for the r?

**Pull request merged**